rskj-core/src/integrationTest/java/pte/PteIntegrationTest.java

```diff
@@ -0,0 +1,270 @@
+                .findFirst()
+                .orElse("");
+
+        Path databaseDirPath = tempDir.resolve("database");
+        String databaseDir = databaseDirPath.toString();
+        String[] baseArgs = new String[]{
+                String.format("-Xdatabase.dir=%s", databaseDir),
+                "--regtest",
+                String.format("-Xrpc.providers.web.http.port=%s", RPC_PORT)
+        };
+        strBaseArgs = String.join(" ", baseArgs);
+        baseJavaCmd = String.format("java %s %s", String.format("-Dlogback.configurationFile=%s", logbackXmlFile), String.format("-Drsk.conf.file=%s", rskConfFile));
+    }
+
+    @Test
+    void whenParallelizableTransactionsAreSent_someAreExecutedInParallel() throws Exception {
+
+        // Given
+
+        // Pre-funded Test Accounts on Regtest
+        List<String> accounts = Arrays.asList(
+                "0xcd2a3d9f938e13cd947ec05abc7fe734df8dd826",
+                "0x7986b3df570230288501eea3d890bd66948c9b79",
+                "0x0a3aa774752ec2042c46548456c094a76c7f3a79",
+                "0xcf7cdbbb5f7ba79d3ffe74a0bba13fc0295f6036",
+                "0x39b12c05e8503356e3a7df0b7b33efa4c054c409",
+                "0xc354d97642faa06781b76ffb6786f72cd7746c97",
+                "0xdebe71e1de41fc77c44df4b6db940026e31b0e71",
+                "0x7857288e171c6159c5576d1bd9ac40c0c48a771c",
+                "0xa4dea4d5c954f5fd9e87f0e9752911e83a3d18b3",
+                "0x09a1eda29f664ac8f68106f6567276df0c65d859",
+                "0xec4ddeb4380ad69b3e509baad9f158cdf4e4681d"
+        );
+
+        Map<String, Response> txResponseMap = new HashMap<>();
+        Map<String, Map<Integer, String>> blocksResponseMap =  new HashMap<>();
+
+        String cmd = String.format(
+                "%s -cp %s/%s co.rsk.Start --reset %s",
+                baseJavaCmd,
+                buildLibsPath,
+                jarName,
+                strBaseArgs);
+
+        // When
+
+        CommandLineFixture.runCommand(
+                cmd,
+                1,
+                TimeUnit.MINUTES,
+                proc -> {
+                    try {
+
+                        // Send bulk transactions
+
+                        Response txResponse = sendBulkTransactions(
+                                accounts.get(0), accounts.get(1), accounts.get(2), accounts.get(3),
+                                accounts.get(4), accounts.get(5), accounts.get(6), accounts.get(7));
+
+                        txResponseMap.put("bulkTransactionsResponse", txResponse);
+
+                        // Await for n blocks to be mined and return them
+
+                        Future<Map<Integer, String>> future = getBlocksAsync();
+
+                        try {
+
+                            blocksResponseMap.put("asyncBlocksResult", future.get());
+
+                        } catch (ExecutionException | InterruptedException e) {
+                            e.printStackTrace();
```

Comment:
I'd expect that `Assertions.fail()` below should print it too, isn't `e.printStackTrace()` just redundant here?